routes/index.js

```diff
@@ -1,21 +1,27 @@
 var express = require("express");
 var router = express.Router();
 var mood = require("../backend/city_tweet_mood");
+var cities = require("../public/assets/cities");
 
 /* GET home page. */
-router.get("/", function (req, res, next) {
+router.get("/", function(req, res, next) {
   res.render("index", {
-    title: "TWITTER MOOD",
+    title: "TWITTER MOOD"
   });
 });
 
-router.post("/mood", function (req, res) {
+router.post("/mood", function(req, res) {
   mood
-    .get_tweets_and_sentiment(req.body.city, req.body.state)
-    .then((tweets) => {
+    .get_tweets_and_sentiment(req.body.coords)
+    .then(tweets => {
       res.render("tweets", { title: "TWITTER MOOD", tweets: tweets });
     })
     .catch(); //TODO: Error handling
 });
 
+router.get("/cities.json", function(req, res) {
```

Comment:
is this used? it looks like the JS above is just grabbing it directly from the assets folder?

Reply:
It is! The ajax call sends a GET request to the server. The route allows the server to serve up the file to the client. You'd get a 404 on this otherwise:
```
// Retrieve city data from JSON file
$(function() {
  cities = $.ajax({
    type: "GET",
    url: "../cities.json",
    async: false
  }).responseJSON;
});
```

Reply:
good point.  Works for me